### ISSUE TITLE
Output Selected Rows - No returned values and no error

### ISSUE THREAD

**Issue by anjavel** [ericrsmith35/DatatableV2#77]
Hi, 

I have an issue similar to an earlier post. When I change the amount in the table and then click 'Opslaan' (meaning next), it returns numberOfRowsEdited = null.

So here I edit the quantity and click 'Opslaan'
![image](https://user-images.githubusercontent.com/116079382/196420857-c9f9e656-b4ef-4d9d-9960-754b0c9f992c.png)

After that the quantity fields are emptied: 
![image](https://user-images.githubusercontent.com/116079382/196421325-1b483392-021e-4e4c-a968-55e7cf8e3bf7.png)

I looked at earlier posts and saw there was an issue with geolocation fields, but this object doesn't have any geolocation fields.
Any idea what could be causing this?

**Comment by ericrsmith35** [comment]
What version are you running?

**Comment by anjavel** [comment]
Version 3.5.0 it seems. Do you recommend installing a new version? 
Somehow the error isn't showing in our sandbox org, only in the test org (also sandbox) and they are running the same version. 
If I install a new version, could that influence current functionalities that use the datatable component?

**Comment by ericrsmith35** [comment]
Everything should be backwards compatible.  Please install the latest version.

**Comment by anjavel** [comment]
Hi Eric, 
Our administrator has uploaded the newest version. It has uploaded a new version of the lightning component: datatable besides the old datatableV2. As all our flows are using datatableV2, I want to check whether datatableV2 will also be updated when a new version is installed? If that is the case, then we are still running into the same issue after uploading the new version.

**Comment by ericrsmith35** [comment]
No, you will need to update the flows you are having issues with to use the new version.
